src/pages/HomeNeedLeadView.tsx

```diff
@@ -58,20 +59,67 @@ const useStyles = makeStyles(() => ({
 }));
 
 const HomeNeedLeadView = () => {
-  // const { data: needsData } = useQuery(GET_NEEDS, {
-  //   variables: {
-  //     request: {
-  //       options: {
-  //         limit: 5
-  //       }
-  //     }
-  //   },
-  //   fetchPolicy: "network-only"
-  // });
-  // const needs = needsData?.getAllNeeds?.needs;
+  const { data: needsData } = useQuery(GET_NEEDS, {
+    variables: {
+      request: {
+        options: {
+          page: 1,
+          limit: 5
+        }
+      }
+    },
+    fetchPolicy: "cache-and-network"
+  });
+
+  const { data: leadsData } = useQuery(GET_LEADS, {
+    variables: {
+      request: {
+        options: {
+          page: 1,
+          limit: 5
+        }
+      }
+    },
+    fetchPolicy: "cache-and-network"
+  });
+
+  const needs = needsData?.getAllNeeds?.needs || needsDummy;
+  const leads = leadsData?.getAllLeads?.leads || leadsDummy;
```

Comment:
Okey but it is going to appear in production now - You should either create a test file where you can use dummy data to mock the graphql query. 
Here you need to change for: ** needsData?.getAllNeeds?.needs || []**

Then we can displa a message  "No needs to display for now" 

And to test that it work you can run the backend and create some needs/leads via the UI :)

Reply:
Sounds good, I'm just going to delete the dummy data because it's just a temporary thing anyway and I already created needs and leads locally through the UI